rel/reltool.config

```diff
@@ -11,7 +11,7 @@
 % the License.
 
 {sys, [
-    {lib_dirs, ["../src"]},
```

Comment:
Why do we still need this file. I thought we are switching to relx?

Reply:
Because `./dev/run` is using `reltool`. I'll look for a solution later, thanks for the review.